server/src/server.js

```diff
@@ -682,6 +697,21 @@ app.post("/api/delete-shot", csrfProtection, function(req, res) {
   });
 });
 
+app.post("/api/disconnect-device", csrfProtection, function(req, res) {
+  if (!req.deviceId) {
+    sendRavenMessage(req, "Attempt to disconnect without login");
+    simpleResponse(res, "Not logged in", 401);
+    return;
+  }
+  disconnectDevice(req.deviceId).then((result) => {
```

Comment:
How does this change the accountId cookie?  It should delete that cookie, otherwise it looks like you'll still have the associated account for the current session.